File: 9-Maps/maps.js
function processData(input) {

  let phoneBook = {},
    arr = input.split("\n");

  for (let i = 1; i <= +arr[0]; i++) {
    let entry = arr[i].split(" ");
    phoneBook[`${entry[0]}`] = `${entry[1]}`;
  }

  for (let j = +arr[0] + 1; j < arr.length; j++) {
    let name = arr[j];

    if (phoneBook[`${name}`]) {
      console.log(`${name}=${phoneBook[`${name}`]}`)
    } else {
      console.log("Not found")
    }
  }
}

File: 9-Maps/maps.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
#include <map>
using namespace std;

int main()
{
  int N;
  cin >> N;

  map<string, int> phoneBook;

  for (int i = 0; i < N; i++)
  {
    string name;
    cin >> name;
    int phone;
    cin >> phone;

    phoneBook[name] = phone;
  }

  string name;

  while (cin >> name)
  {
    if (phoneBook.find(name) == phoneBook.end())
    {
      cout << "Not found" << endl;
    }
    else
    {
      cout << name << "=" << phoneBook[name] << endl;
    }
  }

  return 0;
}
